api/types/constants.go

```diff
@@ -323,6 +325,16 @@ const (
 	OriginCloud = "cloud"
 )
 
+const (
+	// AWSNamespace is used as the namespace prefix for any labels
+	// imported from AWS.
+	AWSNamespace = "aws"
+	// EC2LabelUpdatePeriod is the period for updating EC2 labels.
+	EC2LabelUpdatePeriod = time.Hour
```

Comment:
Let's get away from putting public vars/consts in cases where they're only needed in one place.

In this case, you only use this in `package labels`, so you can move the constant(s) there and unexport them.